Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -899,6 +899,10 @@ private void ThreadUpdateApp(bool automatically, CancellationToken cancellationT
             {
                 _appInfo.Value = _app.RemoteMetaData.GetAppInfo(!automatically, _updateAppCancellationTokenSource.Token);
                 _remoteVersionId.Value = _app.GetLatestVersionId(!automatically, _updateAppCancellationTokenSource.Token);
+                var latestAppVersion =
```

Comment:
Don't use `var` if the type cannot be determined by looking at the right side.

Reply:
done